service/src/main/java/uk/gov/hmcts/reform/fpl/validation/groups/ValidateCaseNumberGroup.java

```diff
@@ -1,5 +1,5 @@
 package uk.gov.hmcts.reform.fpl.validation.groups;
 
 // JSR Validation interface used to validate the C21 Group
-public @interface C21CaseOrderGroup {
+public @interface ValidateCaseNumberGroup {
```

Comment:
Do we need to be more specific with this and have `ValidateFamilyManCaseNumberGroup`?

Reply:
sure, doesn't hurt to be verbose